application/models/SurveyDynamic.php

```diff
@@ -388,12 +388,12 @@ public function getExtendedData($colName, $sLanguage, $base64jsonFieldMap)
         $oFieldMap = json_decode(base64_decode($base64jsonFieldMap));
         $value     = $this->$colName;
 
-        $sFullValue = strip_tags(getExtendedAnswer(self::$sid, $oFieldMap->fieldname, $value, $sLanguage));
+        $sFullValue = getExtendedAnswer(self::$sid, $oFieldMap->fieldname, $value, $sLanguage);
         if (strlen($sFullValue) > 50) {
             $sElipsizedValue = ellipsize($sFullValue, $this->ellipsize_question_value);
             $sValue          = '<span data-toggle="tooltip" data-placement="left" title="' . quoteText($sFullValue) . '">' . $sElipsizedValue . '</span>';
         } else {
-            $sValue          = $sFullValue;
+            $sValue          = CHtml::encode($sFullValue);
```

Comment:
If I don't make error : `getExtendedAnswer` return non stripped value for single choice.

Then if in a single choice question : we have `<strong>Really good</strong>` : it shown `<strong>Really good</strong>` but we need  

- stripped for answer from survey (maybe wit code)
- encoded for user answer

See https://gitlab.com/SondagesPro/coreAndTools/getQuestionInformation/-/blob/master/helpers/surveyColumnsInformation.php#L1004 and https://gitlab.com/SondagesPro/coreAndTools/getQuestionInformation/-/blob/master/helpers/surveyColumnsInformation.php#L1012

Reply:
strip_tags is not a good option here.
It works ok for flatening the answer.
But it doesn't do well when answer options are like  " < 1" .
(strip tags gets confused and produces the reported bug)

Reply:
Yes, but it's not the question : it's about the value return must be different between single answer choice and free text.